README.md

```diff
@@ -74,7 +74,7 @@ shared objects required for the cgo driver.
 Compilation:
 
 ```sh
-CGO_LDFLAGS="-L/path/to/OCS/lib -lsybct_r64 -lsybcs_r64" go build -o cmd ./
+CGO_LDFLAGS="-L/path/to/OCS/lib" go build -o cmd ./
```

Comment:
Clarification: Is it only `OCS` or rather `OCS(_<version-#>)`?

Reply:
OCS_Version - but what `/path/to/OCS` should be is specified in the original text above the change.